### ISSUE TITLE
Http Error (ETIMEDOUT)

### ISSUE THREAD

**Issue by Mendelyeev** [bwp91/homebridge-ewelink#241]
### Details of your setup.
* Homebridge (+ config-ui) - 4.41.0
Raspberry pi
NODE:14.16.1
NPM:7.9.0
Plugin Version: 6.2.1 (issue also occours on 6.2.0)
eWelink Device: Sonoff Mini

### Please paste any relevant logs below.

From yesterday to today my Homebridge setup started to shown an error (The only thing I dida was updated plugin to v6.2.1): 

I also tried to turn on and off trought eWelink app and worked fine. Follow is the debug log:

[19/04/2021, 10:50:15] [eWeLink] ***** Disabling plugin [v6.2.1]. *****
[19/04/2021, 10:50:15] [eWeLink] ***** Error: connect ETIMEDOUT 52.28.137.33:443
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16) {
  errno: -110,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '52.28.137.33',
  port: 443,
  config: [Object],
  request: [Writable],
  response: undefined,
  isAxiosError: true,
  toJSON: [Function: toJSON]
}. *****

```

```

**Comment by bwp91** [comment]
Hi @Mendelyeev 
Looks like a connection problem between your homebridge rpi and the ewelink servers.
Has anything changed with your internet connection?
Do you get this _everytime_ you restart the plugin?

**Comment by Mendelyeev** [comment]
Thanks for your answer.
I noticied that I used normally this plugin today earlier. But now, everytime I get this error.

**Comment by Mendelyeev** [comment]
Ididn't change any setting on my internet connection.

**Comment by bwp91** [comment]
Could you update to beta version 6.2.2-beta.0?
It won't fix this issue but hopefully in the logs will pinpoint more specifically where the error is occurring.
Could you paste the logs from the plugin after updating and restarting Homebridge?

**Comment by Mendelyeev** [comment]
@bwp91, Ben You were right. I discovered tha my ISP is blocking several IP adresses !!! Sorry for make you lose time with this! Ss you requested, I am sending the debug log, anyway:


```
[19/04/2021, 15:12:45] [eWeLink] ***** Disabling plugin [v6.2.2-beta.0]. *****
[19/04/2021, 15:12:45] [eWeLink] ***** Error: connect ETIMEDOUT 52.28.137.33:443
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16) {
  errno: -110,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '52.28.137.33',
  port: 443,
  config: [Object],
  request: [Writable],
  response: undefined,
  isAxiosError: true,
  toJSON: [Function: toJSON]
}. *****
```

**Comment by bwp91** [comment]
Were the any messages before this? Something like

```
An error occurred during ...
```

**Comment by Mendelyeev** [comment]
@bwp91 , Only this:

```
[19/04/2021, 15:10:35] [eWeLink] [v6.2.2-beta.0] initialised. Syncing with eWeLink.
[19/04/2021, 15:10:35] [eWeLink] Sending HTTP login() request.
```

**Comment by bwp91** [comment]
Out of interest if you update to 6.2.2-beta.1 do you get anything extra in the logs?

So can you contact your ISP to unblock this IP?

**Comment by Mendelyeev** [comment]
I already contacted ISP to unblock theses addresses. I found 3 that are constantly blocked:
```
52.28.137.33
35.158.225.153
3.125.15.60
```
Can you tell me if have another(s) ip address(es) that needed to be unblocked?

Following is the debug log from 6.2.2-beta.1:

```
[19/04/2021, 15:49:11] [eWeLink] An error occurred during http login().
[19/04/2021, 15:49:11] [eWeLink] ***** Disabling plugin [v6.2.2-beta.1]. *****
[19/04/2021, 15:49:11] [eWeLink] ***** Error: connect ETIMEDOUT 3.125.15.60:443
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16) {
  errno: -110,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '3.125.15.60',
  port: 443,
  config: [Object],
  request: [Writable],
  response: undefined,
  isAxiosError: true,
  toJSON: [Function: toJSON]
}. *****
```

**Comment by Mendelyeev** [comment]
Now worked 👍🏻. And seems that my ISP still blocking those IP addresses. Did you fix anything?

**Comment by bwp91** [comment]
Nope i haven’t done anything.

Could have just been some intermittent issue somewhere between your pi and the server is was trying to connect to.

**Comment by Mendelyeev** [comment]
Ben, I've got one debug log that said something like API was changed. Sorry but I truncated log file after that and cannot paste here...

**Comment by bwp91** [comment]
Most likely a log saying 
`New API host received`?
This is expected if you’re not based in Europe

**Comment by Mendelyeev** [comment]
Yep! I'm from Brazil.
